fastai/tabular/learner.py

```diff
@@ -1,17 +1,21 @@
-# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/43_tabular.learner.ipynb (unless otherwise specified).
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/nbs/43_tabular.learner.ipynb (unless otherwise specified).
 
 __all__ = ['TabularLearner', 'tabular_learner']
 
 # Cell
 from ..basics import *
 from .core import *
 from .model import *
+from .data import *
 
 # Cell
 class TabularLearner(Learner):
     "`Learner` for tabular data"
-    def predict(self, row):
-        "Predict on a Pandas Series"
+    def predict(
+        self,
+        row:pd.Series, # Features to be predicted on
```

Comment:
`# Input values for the prediction` maybe.  See if reviewer2 has a better idea than me for wording there :)

Reply:
```suggestion
        row:pd.Series, # Features to be predicted
```
@kevinbird15 let me know if you agree this sounds better!